src/flint/fq_nmod.jl

```diff
@@ -102,10 +102,7 @@ is_unit(a::fqPolyRepFieldElem) = ccall((:fq_nmod_is_invertible, libflint), Bool,
                      (Ref{fqPolyRepFieldElem}, Ref{fqPolyRepField}), a, a.parent)
 
 function characteristic(a::fqPolyRepField)
-   d = ZZRingElem()
-   ccall((:__fq_nmod_ctx_prime, libflint), Nothing,
-         (Ref{ZZRingElem}, Ref{fqPolyRepField}), d, a)
-   return d
+   return a.n
```

Comment:
```suggestion
   return ZZ(a.n)
```
Lets keep it an `fmpz` as before.